packages/framework/src/Command/CreateApplicationDirectoriesCommand.php

```diff
@@ -38,26 +39,19 @@ class CreateApplicationDirectoriesCommand extends Command
     private $uploadedFileDirectoryStructureCreator;
 
     /**
-     * @var string
-     */
-    private $projectDir;
-
-    /**
-     * @var string
+     * @var \Symfony\Component\DependencyInjection\ContainerInterface
      */
-    private $webContentDirName;
+    private $container;
 
     /**
-     * @param string $projectDir
-     * @param string $webContentDirName
+     * @param \Symfony\Component\DependencyInjection\ContainerInterface $container
      * @param \League\Flysystem\FilesystemInterface $filesystem
      * @param \Symfony\Component\Filesystem\Filesystem $localFilesystem
      * @param \Shopsys\FrameworkBundle\Component\Image\DirectoryStructureCreator $imageDirectoryStructureCreator
      * @param \Shopsys\FrameworkBundle\Component\UploadedFile\DirectoryStructureCreator $uploadedFileDirectoryStructureCreator
      */
     public function __construct(
-        string $projectDir,
-        string $webContentDirName,
+        ContainerInterface $container,
```

Comment:
I don't really like `ContainerInterface` here and relay on some parameters, that may not be specified.
Is possible to pass just `local` and `main` directories?